### PR TITLE
Implement ES6 syntax and modules to recreate the Awesome Books

### PR DESCRIPTION
**Pull request by abreham1222**
-Recreated the HTML structure of the Awesome Books app
-Implemented the same functionality but using ES6 syntax, including arrow functions, let, const, classes and more.
-Split the app into modules (i.e. Book, Library, BookForm, and Router modules).
-Added styles so it looks like the required project
-Added the Luxon package as a module

### CODE REVIEW

**Review by addod19 — COMMENTED** [review]
### Hi @abreham1222, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failed the linter checks

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    <p>You can reach out to us!</p>
+    <ul>
+      <li>Our e-mail mail@mail.com</li>
+      <li>Our Phone number: 0043586534422</li>
+      <li>Our address: Street name 22, 84503 City. Country.</li>
+    </ul>
+  </section>
+  <footer>
+    Copyright Awesome Books 2022
+  </footer>
```
Comment (TedLivist) [inline index.html:50]: - [ ] **[OPTIONAL]** I would recommend you make your footer stick to the bottom of the page.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @abreham1222,

✔️ Books can be added and removed properly

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @abreham1222 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉👏🥇✨

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — COMMENTED** [review]
## Status: Approved :heavy_check_mark:

Hi Abreham,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾Congratulations!  🎉🌟 

## Highlights 👍
- App is running as intended
- Luxon library properly implemented
- Code followed ES6 syntax

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**